agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1667,3 +1667,18 @@ void ap_manager_thread::send_steering_return_status(beerocks_message::eActionOp_
     }
     return;
 }
+
+void ap_manager_thread::remove_client_from_disallowed_list(
```

Comment:
unnecessary since the "remove" helper function is the only one that is used more than once. 

About the naming ,
 `allow_client` is misleading since removing a client from this list **doesn't** always mean the client will be automatically allowed it could be added again with different parameters (timeout, target bssid...)

Reply:
Even if it is only used once - it still make sense to put in a helper function simply because its more readable.
Naming - you choose then.